ibm/mas_devops/roles/ocp_setup_aws_efs/README.md

```diff
@@ -0,0 +1,10 @@
+ocp_setup_aws_efs
+==================
+
+This role provides support to configure AWS EFS based storageclass into Openshift cluster.
```

Comment:
Can you elaborate on what this means?  What are potential uses for the AWS EFS storageclass within MAS.  Also please document variables, e.g. see examples and a sample playbook for the role ... see https://github.com/ibm-mas/ansible-devops/blob/master/ibm/mas_devops/roles/mongodb/README.md for example of the expected format of the README